packages/eslint-config/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@frontside/eslint-config",
+  "version": "1.0.0",
+  "description": "Shared eslint config @thefrontside",
+  "main": "index.js",
+  "repository": "https://github.com/thefrontside/eslint-config",
+  "author": "Frontside Engineering <engineering@frontside.com>",
+  "license": "MIT",
+  "private": false,
+  "keywords": ["eslint", "eslintconfig"]
+}
```

Comment:
Should we depend on `@typescript-eslint/eslint-plugin` and `eslint-plugin-prefer-let`?

Reply:
How do you mean?

Reply:
@minkimcello meaning that because we consume rules from those packages, maybe we should have those packages in `package.json#dependencies`

Reply:
Added the dependencies to the `eslint-config` package so now the projects can just install `@frontside/eslint-config` and won't need to install the dependencies separately.

![Screen Shot 2020-07-01 at 10 36 06 AM](https://user-images.githubusercontent.com/29791650/86256816-c5658d00-bb86-11ea-9e99-4cf3827f7360.png)

Reply:
Is this what we were going for?